internal/provider/resources/install/ssh/aws.go

```diff
@@ -71,6 +74,12 @@ Installing SSH allows you to manage access to your servers on AWS.`,
 				MarkdownDescription: `If present, AWS instances will be grouped by the value of this tag. Access can be requested, in one request, to all instances with a shared tag value`,
 				Optional:            true,
 			},
+			"is_sudo_enabled": schema.BoolAttribute{
+				MarkdownDescription: `If true, users will be able to request sudo access to the instances`,
```

Comment:
Question: Do we use sentences for descriptions?

```suggestion
				MarkdownDescription: `If true, users will be able to request sudo access to the instances.`,
```

Reply:
We don't none of the other resources have a period at the end for the descriptions 🤔